service/project.go

```diff
@@ -733,3 +735,25 @@ func (uis *UIServer) projectEvents(w http.ResponseWriter, r *http.Request) {
 	}{id, uis.GetCommonViewData(w, r, true, true)}
 	uis.render.WriteResponse(w, http.StatusOK, data, "base", template, "base_angular.html", "menu.html")
 }
+
+func verifyAliasExists(alias, projectIdentifier string, newAliasDefinitions []model.ProjectAlias, deletedAliasDefinitionIDs []string) (bool, error) {
+	if len(newAliasDefinitions) > 0 {
+		return true, nil
+	}
+
+	existingAliasDefinitions, err := model.FindAliasInProject(projectIdentifier, alias)
+	if err != nil {
+		return false, errors.Wrap(err, "error checking for existing aliases")
+	}
+
+	remainingAliases := false
+	for _, a := range existingAliasDefinitions {
+		// only consider aliases that won't be deleted
+		if !util.StringSliceContains(deletedAliasDefinitionIDs, a.ID.Hex()) {
+			remainingAliases = true
```

Comment:
Rather than having a variable remainingAliases here, can you just return `true, nil` at 753 and and `false, nil` at 758?

Reply:
done